falcon/middleware.py

```diff
@@ -75,7 +82,9 @@ def __init__(
                 )
         self.allow_credentials = allow_credentials
 
-    def process_response(self, req: Request, resp: Response, resource, req_succeeded):
+    def process_response(
+        self, req: Request, resp: Response, resource: Resource, req_succeeded: bool
```

Comment:
resource is an object, I think it's ok to just type it as object

Reply:
Changed.